devlib/platform/gem5.py

```diff
@@ -127,6 +129,11 @@ def _start_interaction_gem5(self):
             # telnet connection.
             self.stderr_filename = f
 
+            # Enable power modelling if required
+            if self.power_models_dir:
```

Comment:
OK, it seems there is a consensus that this is too specific to a particular script to be generally useful, and there isn't much more that makes sense beyond the existing `gem5args`. I'm closing this pull.